docs/tutorials/migrations/homebrew.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Homebrew
+description: Using Flox to replace or augment Homebrew
+---
+
+# Homebrew migration guide
+
+Flox is an environment and package manager all in one. Using Flox, you can create discrete environments containing packages from the Flox Catalog.
+
+Flox can replace [Homebrew](https://brew.sh) entirely, or they can be used together.
+
+This guide explains how to introduce Flox into environments where Homebrew is currently being used, either as a replacement or an addition. It introduces new concepts and proposes a basic procedure for mapping packages.
+
+## Why you might want to migrate
+Homebrew does a great job, and has been loved as the “missing package manager” for a generation of macOS users. But there are a few reasons you might consider moving to Flox:
+
+* **You need a cross-platform package manager.** Flox works on both macOS and Linux - x86 and ARM - allowing you to define a set of packages that works the same everywhere.
+* **You need a virtual package manager** Flox allows you to create as many environments as you need, each with a different set of packages.
+* **You need reproducible environments.** Flox environments are defined in a TOML [manifest][manifest_concept] that can be easily shared, or managed alongside code.
+* **You need a broader set of software.** Homebrew has a sizeable collection of packages, but the Flox Catalog is based on Nixpkgs, the largest collection of packages in the world.
+* **You need older versions of software.** The Flox Catalog contains historical versions of each of its packages - in many cases, going back decades.
```

Comment:
> going back decades

I don't think that's true. Our catalog only goes back about 3 years as far as I'm aware. The Nix binary cache may have entries that old, but they aren't really accessible through our catalog.